### ISSUE TITLE
Question about training with only AVA-AVD

### ISSUE THREAD

**Issue by zaocan666** [zcxu-eric/AVA-AVD#2]
Hi, this is indeed fantastic work!
I have some problems with training on the AVA-AVD dataset only.
I trained for 28000 iterations, getting training loss of about 3.18. However, the DER is as high as 53 in the validation and test set (I have tried different thresholds). While the DER reported in the paper should be 27.
Should I train more iterations or adjust the learning rate or something? It would be very kind of you to supply the configuration of training only on the AVA-AVD dataset.

**Comment by zcxu-eric** [comment]
Thanks for your interest. I have checked my training log, after 28000 iterations, the DER should be around 26.79. During training, this codebase will search for the best threshold. 

FYI, this is the training command I used for this experiment `python mmsc_exp/run.py config=projects/token/mix.yaml datasets=ava,avaavd model=token run_type=train training.batch_size=8 env.save_dir=./save/token/ava`. The configs have already been included in this codebase.

Did you successfully initialize the weights of the visual and audio encoders from pretrained models?
